challenges/2022-11-08-common-elements/solution.js

```diff
@@ -0,0 +1,27 @@
+// # Common Elements
+
+// Create a function that takes two "sorted" arrays of numbers and returns an array of numbers which are common to boththe input arrays.
+
+// ## Business Rules/Errata
+
+// - You can expect the arrays to be sorted
+// - Arrays do not have to be the same length
+// - This problem should be solved in O(n+m) time where n is the length of the first input array and m is the length of the second input array
+
+// ## Examples
+
+// ```
+// commonElements([-1, 3, 4, 6, 7, 9], [1, 3]) -> [3]
+// commonElements([1, 3, 4, 6, 7, 9], [1, 2, 3, 4, 7, 10]) -> [1, 3, 4, 7]
+// commonElements([1, 2, 2, 2, 3, 4, 5], [1, 2, 4, 5]) -> [1, 2, 4, 5]
+// commonElements([1, 2, 3, 4, 5], [10, 12, 13, 15]) -> []
```

Comment:
Should the result of `commonElements([1, 1], [1, 1])` be `[1]` or `[1, 1]` (i.e., should elements repeated in both arrays be repeated in the output)?